shipcat_filebacked/src/load.rs

```diff
@@ -155,7 +161,10 @@ async fn read_from<T: DeserializeOwned>(path: &PathBuf) -> Result<T> {
     if data.is_empty() {
         bail!("Manifest file {} is empty", path.display());
     }
-    Ok(serde_yaml::from_str(&data)?)
+    match serde_yaml::from_str(&data) {
+        Err(e) => bail!("Manifest file {} did not parse: {}", path.display(), e),
```

Comment:
Could you say that it's invalid YAML (as that's useful for debugging, as you know to look for a basic syntax error)?

e.g., 

```suggestion
        Err(e) => bail!("Manifest file {} did not parse as YAML: {}", path.display(), e),
```